gobusterdir/gobusterdir.go

```diff
@@ -87,11 +89,11 @@ func (d GobusterDir) ResultToString(g *libgobuster.Gobuster, r *libgobuster.Resu
 	// Prefix if we're in verbose mode
 	if g.Opts.Verbose {
 		if g.Opts.StatusCodesParsed.Contains(r.Status) {
-			if _, err := fmt.Fprintf(buf, "Found: "); err != nil {
+			if _, err := fmt.Fprintf(buf, Sprintf(Green("Found: "))); err != nil {
 				return nil, err
 			}
 		} else {
-			if _, err := fmt.Fprintf(buf, "Missed: "); err != nil {
+			if _, err := fmt.Fprintf(buf, Sprintf(Red("Missed: "))); err != nil {
```

Comment:
I think I'd rather see `Missed` outputted in orange or yellow. It's not exactly an error.